yt/frontends/athena/io.py

```diff
@@ -18,8 +26,8 @@ class IOHandlerAthena(BaseIOHandler):
 
     def _field_dict(self, fhandle):
         keys = fhandle["field_types"].keys()
-        val = fhandle["field_types"].keys()
-        return dict(zip(keys, val))
+        val = fhandle["field_types"].values()
```

Comment:
this is a case where the original code just *looks* wrong, but it's been there for 12 years so either it's not really a bug (but it should have a comment), or it's actually not used by anyone...

Reply:
I don't see any references to it -- this may be vestigial.

Reply:
you're right. And it's private anyway so let me just remove it now.